### PR TITLE
makes santa on antag panel

### PR DESCRIPTION
**Pull request by boodaliboo**
antag panel santa is now

### CODE REVIEW

**Comment by boodaliboo** [comment]
is good

**Comment by boodaliboo** [comment]
woops

File: code/modules/antagonists/santa/santa.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /datum/antagonist/santa
 	name = "Santa"
-	show_in_antagpanel = FALSE
+	show_in_antagpanel = True
```
Comment (alexkar598) [inline code/modules/antagonists/santa/santa.dm]:
```suggestion
	show_in_antagpanel = TRUE
```

**Review by alexkar598 — CHANGES_REQUESTED** [review]
More shitty coders

**Review by alexkar598 — APPROVED** [review]

**Pull request merged**